Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -59,6 +59,7 @@ def runSelected(opt):
                      135.4, #Run 2 Zee ttbar
                      10021.0, #2017 tenmu
                      10024.0, #2017 ttbar
+                     10424.0, #2017 NewPix ttbar
```

Comment:
10424.0 was a 2023 workflow for some time, though a re-numbering has recently occurred. Why not use a number from 10246-10309, which hasn't been used before?